### DISCUSSION TITLE
How to reset password of Velaux ？ DaLao，Please help me.

### DISCUSSION THREAD

**Discussion by ormosias** [kubevela/kubevela#6116] [Q&A]
I have reinstalled velaux

![image](https://github.com/kubevela/kubevela/assets/90101814/61a0dc76-d0fd-4045-a426-1c2a14a24a40)

**Comment by wonderflow** [comment]
https://kubevela.io/docs/developers/references/devex/faq#how-to-change-the-admin-password-of-velaux
